pkgs/applications/networking/instant-messengers/rambox/pro.nix

```diff
@@ -46,6 +13,23 @@ stdenv.mkDerivation rec {
     categories = "Network;";
   };
 
+  src = fetchurl {
+    url = "https://github.com/ramboxapp/download/releases/download/v${version}/RamboxPro-${version}-linux-x64.AppImage";
+    name = "${name}.AppImage";
+    sha256 = "18383v3g26hd1czvw06gmjn8bdw2w9c7zb04zkfl6szgakrv26x4";
+  };
+
+  appimageContents = appimageTools.extract { inherit name src; };
+in appimageTools.wrapType2 rec {
+    inherit name src;
+
+    extraInstallCommands = ''
+      mkdir -p $out/share/applications $out/share/icons/hicolor/256x256/apps
+      cp ${desktopItem}/share/applications/* $out/share/applications
+      cp ${appimageContents}/usr/share/icons/hicolor/256x256/apps/ramboxpro.png $out/share/icons/hicolor/256x256/apps/ramboxpro.png
+    '';
+  };
```

Comment:
There still seems to be some eval issues, or GitHub syntax highlighting is wrong (but I think this very unlikely).